src/common/ownsql.cpp

```diff
@@ -43,9 +43,7 @@ namespace OCC {
 
 Q_LOGGING_CATEGORY(lcSql, "nextcloud.sync.database.sql", QtInfoMsg)
 
-SqlDatabase::SqlDatabase()
-{
-}
+SqlDatabase::SqlDatabase() = default;
```

Comment:
I would prefer this inside the class definition. Any objections?

Reply:
I'd rather not. :-)

I prefer it when moving from `= default` to own implementation (or back) doesn't incur a recompile in the dependencies. Moving those in the header leak that implementation detail and will lead to recompiles.